### ISSUE TITLE
cyto_plot_gating_scheme fails when OR gates are contained in the gating template

### ISSUE THREAD

**Issue by danychauvin** [DillonHammill/CytoExploreR#222]
**Describe the bug**
When trying to use cyto_plot_gating_scheme on my data, using a gating template containing a OR gate, nothing is plotted, and I get the following message:
```
Only NOT boolean gates are supported.
Error in FUN(X[[i]], ...) : NA does not exist in this GatingHierarchy!
```

**To Reproduce**
Apply a gating template to your dataset, with an OR gate. Try to run cyto_plot_gating_scheme.

**Expected behavior**
It should plot all gates that are not OR gates.

**Desktop (please complete the following information):**
 - Linux
 - R 4.3.0
 - cytoexploreR 1.1.0

Let me know if you need more info. Thank you for all the work. This package is fantastic work!

Best,
Dany

**Comment by rwbaer** [comment]
Of course, the error message says, "Only NOT boolean gates are supported.
Error in FUN(X[[i]], ...) : NA does not exist in this GatingHierarchy!"

AND, OR, XOR, and NOT are all boolean gates. Of these, only "NOT" is supported per the error message. 

Thus, this should not be classified as a "bug".  It could remain as a reasonable "feature request".
